README.md

```diff
@@ -43,8 +36,9 @@ $ npm install
 $ gulp
```

Comment:
Update this and other gulp commands to `npm run [command]`, `gulp` only works if you have it installed globally.